scripts/test_emod3d.sh

```diff
@@ -79,9 +83,9 @@ then
 fi
 
 # EMOD3D is not considered to be completed if:
```

Comment:
I think this is easier to read / understand if we invert the logic. 'EMOD3D is considered to be complete if'